python/cucim/src/cucim/skimage/filters/thresholding.py

```diff
@@ -353,6 +353,8 @@ def threshold_otsu(image=None, nbins=256, *, hist=None):
     -----
     The input image must be grayscale.
     """
+    from skimage.filters import threshold_otsu as threshold_otsu_cpu
```

Comment:
Would it make sense to move this `import` to the top-level? We could add an `_` before it if we are concerned with users thinking this is a public API

Reply:
Okay, I went ahead and moved all these up top